stytch/config/config.go

```diff
@@ -1,6 +1,6 @@
 package config
 
-const APIVersion = "5.14.1"
+const APIVersion = "5.14.0"
```

Comment:
Our version numbers never go backwards, especially since v5.14.1 was already released.

```suggestion
const APIVersion = "5.14.2"
```